src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -148,6 +151,16 @@ public async Task<IActionResult> GetExportStatusById(string idParameter)
             return exportActionResult;
         }
 
+        [HttpDelete]
+        [Route(KnownRoutes.ExportJobLocation, Name = RouteNames.CancelExport)]
+        [AuditEventType(AuditEventSubType.Export)]
+        public async Task<IActionResult> CancelExport(string id)
```

Comment:
`id` needs to be renamed to `idParameter` to match the `KnownRoutes.ExportJobLocation` template. Else the parameter value will be null

Reply:
Yeah I missed this one with the merge. Will fix it.